### ISSUE TITLE
Unable to download chrome webdriver

### ISSUE THREAD

**Issue by Mo0rBy** [serenity-bdd/serenity-core#3292]
### What happened?

I am attempting to run FE tests locally before trying to get them running in a CI/CD context. I've updated the pom.xml to use the latest version available (4.0.15) and when I try to run an FE test I get the following error:
```
WARNING: There was an error managing chrome (error sending request for url (https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/118.0.5993.70/mac-arm64/chromedriver-mac-arm64.zip): error trying to connect: tcp connect error: Operation timed out (os error 60)); using driver found in the cache
```
Then the tests try and use whatever webdriver I have cached somewhere which (surprise surprise) doesn't work. I had a quick look at the [Chrome for Testing dashboard](https://googlechromelabs.github.io/chrome-for-testing/) and I can see that the URL in the terminal does not match the URL on that site. The difference is at the end of the url (`mac-arm64/chromedriver-mac` instead of `mac-arm64/chrome-mac`)

In my terminal: `...../118.0.5993.70/mac-arm64/chromedriver-mac-arm64.zip`

On the Chrome for Testing dashboard: `......./118.0.5993.70/mac-arm64/chrome-mac-arm64.zip`

Screenshot of the Chrome for Testing dashboard as proof:
<img width="1587" alt="Screenshot 2023-10-19 at 15 55 42" src="https://github.com/serenity-bdd/serenity-core/assets/88185573/3553b80b-649a-4412-a05d-b0a996a548f3">

I'm assuming this is something that needs to be changed in the WebDriverManager package, or Serenity needs to be updated to use the version of this package that has this correction in.

I haven't got a clue when this URL change will have occurred as the last time I look at our testing repository was 2 months ago, and it was working then.

Are there any recommendations on how to get this working again? I believe the best thing would be to add the WebDriverManager as a dependency and use my own configured webdriver instead of what Serenity autodownloads and configures.

### Serenity BDD version

4.0.15

### JDK version

openjdk version "17.0.6" 2023-01-17

### Execution environment

Macbook Air M1, macOS Ventura 13.6

Chrome (v118, but trying to get anything working)

### How to reproduce the bug.

Write an FE test (I guess it could be super simple and just attempt to search something on Google)
Configure a chrome webdriver in `serenity.conf` or `serenity.properties`
Run the test and observe the failure

### How can we make it happen?

Add it to the Serenity BDD backlog and wait for a volunteer to pick it up

**Comment by wakaleo** [comment]
Serenity no longer uses WebDriverManager or automatic driver downloads - that's all done by Selenium now.

**Comment by wakaleo** [comment]
No, Serenity pretty much always uses the latest versions. I think this is a Selenium issue, not a Serenity one.
